packages/documentation/src/content/docs/integration/deployment/services/backend-service.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Backend service
+---
+
+import { LinkOut } from '@interledger/docs-design-system'
+
+Rafiki’s <LinkOut href="https://github.com/interledger/rafiki/tree/main/packages/backend">`backend`</LinkOut> service is the software’s main service for handling business logic and external communication. The service is responsible for:
+
+- Exposing the endpoints of the [Open Payments APIs](#open-payments) for public clients to perform account management tasks. This is the external communication piece.
```

Comment:
I wouldn't call it "account management" but rather payment initiation and information lookup